provisioning/miniprov/hare_mp/main.py

```diff
@@ -144,6 +147,24 @@ def get_server_type(url: str) -> str:
         logging.error('Cannot get server type (%s)', error)
         return 'unknown'
 
+def is_mkfs_required(url: str) -> bool:
+    try:
+        provider = ConfStoreProvider(url)
+        machine_id = provider.get_machine_id()
+        # Query results into a list of keys where, Const.SERVICE_MOTR_IO
+        # is found as one of the services under `services`.
+        # e.g.
+        # ['node>aaa120a9e051d103c164f605615c32a4>components[1]>services[0]', 
+        # 'node>bbb340f79047df9bb52fa460615c32a5>components[1]>services[0]',
+        # 'node>ccc8700fe6797ed532e311b0615c32a7>components[1]>services[0]']
+        # We try to find this node's machine id in the query result.
+        services = provider.search_val('node' , 'services',
+                                          Const.SERVICE_MOTR_IO.value)
+        check_id = re.compile(f".*{machine_id}.*")
+        return any(check_id.match(service) for service in services)
+    except Exception as error:
+        logging.error('Cannot get stage type (%s)', error)
```

Comment:
It's not an error rather a warning. 
```suggestion
        logging.warn('Failed to get pod type (%s). Current stage will '
                     'be assumed as not required by default', error)

```